### PR TITLE
Mapped terms for bulk Atlas

### PR DESCRIPTION
**Pull request by anjaf**
Mapped terms from bulk Atlas for January release. 
I went through the "REQUIRES_CURATION" file and focussed on the "new_lines" terms from the that are not in previous run.

### CODE REVIEW

File: zoomage_report.CURATED.tsv [hunk 1]
```diff
@@ -55965,3 +55965,225 @@ cell type	normal human epidermal keratinocyte	http://purl.obolibrary.org/obo/CL_
...
+cell type	human pericytes from placenta (hPC-PL)	http://www.ebi.ac.uk/efo/EFO_0005024	E-GEOD-26284	SRR534317	Anja Fullgrabe	13/01/21 18:40
+cell type	human skeletal muscle myoblast (HSMM)	http://purl.obolibrary.org/obo/CL_0000515	E-GEOD-26284	SRR307932	Anja Fullgrabe	13/01/21 18:40
+cell type	human umblical vein endothelial cells (HUVEC)	http://www.ebi.ac.uk/efo/EFO_0002451	E-GEOD-26284	SRR307906	Anja Fullgrabe	13/01/21 18:40
+cell type	intestinal epithelial cells (IECs)	http://purl.obolibrary.org/obo/CL_0002563	E-GEOD-55356	GSM1334671	Anja Fullgrabe	13/01/21 18:40
+cell type	CD4 T cell	http://purl.obolibrary.org/obo/CL_0000624	E-GEOD-14926	GSM372800	Anja Fullgrabe	13/01/21 18:40
+cell type	CD8 T cell	http://purl.obolibrary.org/obo/CL_0000625	E-GEOD-14926	GSM372809	Anja Fullgrabe	13/01/21 18:40
+cell type	CD8 T cell	http://purl.obolibrary.org/obo/CL_0000625	E-GEOD-53358	GSM1290025	Anja Fullgrabe	13/01/21 18:40
+cell type	CD8 T cell	http://purl.obolibrary.org/obo/CL_0000625	E-GEOD-58078	GSM1400450	Anja Fullgrabe	13/01/21 18:40
+cell type	CD8 T cell	http://purl.obolibrary.org/obo/CL_0000625	E-MTAB-7462	ERR3458886	Anja Fullgrabe	13/01/21 18:40
```
Comment (ngeorgeebi) [inline zoomage_report.CURATED.tsv:56003]: @anjaf I might be being a bit mean here, but why not map this to the previous EFO term you used for CD8 positive T lymphocyte? https://www.ebi.ac.uk/ols/ontologies/efo/terms?short_form=EFO_0004219

File: zoomage_report.CURATED.tsv [hunk 2]
```diff
@@ -55965,3 +55965,225 @@ cell type	normal human epidermal keratinocyte	http://purl.obolibrary.org/obo/CL_
...
+developmental stage	FL.00 first flower(s) open stage	http://purl.obolibrary.org/obo/PO_0007026	E-MTAB-8628	ERR3773822	Anja Fullgrabe	13/01/21 18:40
+developmental stage	third instar larvae stage	http://www.ebi.ac.uk/efo/EFO_0002682	E-GEOD-62008	GSM1518370	Anja Fullgrabe	13/01/21 18:40
+diet	high-fat diet	http://www.ebi.ac.uk/efo/EFO_0002757	E-MTAB-7755	HFD_104:biotin	Anja Fullgrabe	13/01/21 18:40
+disease	Adenomyosis	http://www.ebi.ac.uk/efo/EFO_1001757	E-GEOD-7307	GSM176257	Anja Fullgrabe	13/01/21 18:40
+disease	Anaplastic (Malignant) Meningioma	http://www.ebi.ac.uk/efo/EFO_1000082	E-MTAB-4999	ctrl5	Anja Fullgrabe	13/01/21 18:40
+disease	Biliary atresia	http://www.orpha.net/ORDO/Orphanet_30391	E-GEOD-15235	GSM380382	Anja Fullgrabe	13/01/21 18:40
+disease	Corticobasal degeneration	http://www.orpha.net/ORDO/Orphanet_278	E-PROT-58	ppb.iBAQ.UniPenn_Corticobasal.degeneration.2	Anja Fullgrabe	13/01/21 18:40
+disease	Genetic frontotemporal degeneration with dementia	http://www.orpha.net/ORDO/Orphanet_276061	E-PROT-58	ppb.iBAQ.UniPenn_Frontotemporal.dementia.with.parkinsonism.linked.to.chromosome.17_2	Anja Fullgrabe	13/01/21 18:40
```
Comment (ngeorgeebi) [inline zoomage_report.CURATED.tsv:56046]: for disease terms did we decide on an ontology? I thought we weren't using Orphanet as a source in general

File: zoomage_report.CURATED.tsv (hunk 1 above)
Comment (anjaf) [reply zoomage_report.CURATED.tsv:56003]: I never liked this term "CD8-Positive T-Lymphocytes" (http://www.ebi.ac.uk/efo/EFO_0004219) much as it is sort of out of place in the ontology hierarchy (under regulatory T cell), has no equivalent "CD4-postive T-lymphocyte" and is not following EFO style (plural, upper case). 
I'm not an expert on lymphocytes. Is there anything that speaks against matching to CL_0000625?

File: zoomage_report.CURATED.tsv (hunk 2 above)
Comment (anjaf) [reply zoomage_report.CURATED.tsv:56046]: This term is also in EFO.

File: zoomage_report.CURATED.tsv (hunk 1 above)
Comment (anjaf) [reply zoomage_report.CURATED.tsv:56003]: You're right, this was inconsistent with the older term mappings. "CD8 T cells" and "CD8-positive T-Lymphocyte" are now both mapped to http://purl.obolibrary.org/obo/CL_0000625.

**Pull request merged**